docs/installation.rst

```diff
@@ -0,0 +1,94 @@
+Installation
+============
+
+Installing GeoCAT-viz via Conda
+-------------------------------
+
+The easiest way to install GeoCAT-viz is using
+`Conda <http://conda.pydata.org/docs/>`_::
+
+    conda create -n geocat -c conda-forge -c ncar geocat-viz
```

Comment:
I'd like to avoid a confusion here (and maybe we need to make things clearer in the Contributor's guide and repo-specific docs):

"Installation" (i.e. installation.rst) is aimed at guiding the "end-user" to have an environment for just using the GeoCAT toolkits (i.e. no development). That is the reason, regardless of the toolkit, the installation.rst guides them to create an environment called "geocat" and to install the corresponding toolkit (GeoACT-viz in this case). This could provide to the end-users to have varying GeoCAt toolkits under the same "geocat" environment. 

However, "conda_environment.yml" (or sometimes just "environment.yml") under the root directory or another folder (we named some of them "build_envs" and the newer of them "ci" to more comply with the Pangeo stack) is for the developer's convenience (i.e. for continuous integration purposes). That is the reason the conda environment being created with this file is named "<toolkit name>_build". 

I hope that is helpful.

I'd suggest to get rid of the duplicate files here (root/conda_environment.yml and build_envs/environment; my preference would be to keep the folder and remove the file in the root dir) as well as to rename "build_envs" to "ci" if you decide on keeping that folder.